### DISCUSSION TITLE
Let users control and see PSU temp and fan speed

### DISCUSSION THREAD

**Discussion by Bluscream** [Rem0o/FanControl.Releases#788] [Ideas]
Its the only fan in my PC that i dont have control over

**Comment by Rem0o** [comment]
PSU fans are not interfaced to be controlled by software.

**Reply by Bluscream** [reply]
Could i solder a standard RPM connector to it?

**Reply by Rem0o** [reply]
You tell me. FanControl interfaces motherboard headers. Anything you plug into it is up to you.

**Comment by UrmasK222** [comment]
If you ask this simple question I recommend you do not open your PSU. You might even kill yourself. PSU capacitors "collect" voltage which, depending on your PSU and the way you handle it, can be fatal.
Just buy a PSU with an active fan control built in and you are good to go. That means that for example under 50% of load the PSU fan doesn't work at all. And quality PSU-s are one of the most silent components in your system anyway. You wouldn't even notice it.

Br
Urmas

**Reply by Rem0o** [reply]
^ This. PSU fans are almost always super low-rpm anyway, you probably can't even hear it.
